### PR TITLE
[Check-in] Add voPersonID attribute

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR introduces the new `voPersonID` attribute and updates the scope-claims mapping.
Also, some code style formatting to maintain the markdown clean.

### CODE REVIEW

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -120,8 +120,8 @@ provide the following information to connect your service to EGI Check-in:
 1. Name of the service (in English and optionally in other languages supported
    by the service)
 1. Short description of the service
-1. Site (URL) for localised information about the service; the content found
-   at the URL SHOULD provide more complete information than what provided by the
+1. Site (URL) for localised information about the service; the content found at
+   the URL SHOULD provide more complete information than what provided by the
```
Comment (gwarf) [inline content/en/providers/check-in/sp/_index.md:124]:
```suggestion
1. Site (URL) for localised information about the service; the content found at
   the URL SHOULD provide more complete information than what is provided by the
```

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -215,8 +215,11 @@ bundle to connected Service Providers without administrative involvement,
 subject to user consent. The following attributes constitute a minimal subset of
 the R&S attribute bundle:
 
-- Persistent, non-reassignable, non-targeted, opaque, globally unique EGI user
-  ID (`eduPersonUniqueId`); this is always scoped `@egi.eu`
+- Community User Identifier (CUID) which is a globally unique, opaque,
+  persistent and non-reassignable identifier that identifies the user
```
Comment (gwarf) [inline content/en/providers/check-in/sp/_index.md:219]:
```suggestion
  persistent and non-reassignable identifier identifying the user
```

File: content/en/providers/check-in/sp/_index.md (hunk 1 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: Done

File: content/en/providers/check-in/sp/_index.md (hunk 2 above)
Comment (nikosev) [reply content/en/providers/check-in/sp/_index.md]: Done

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by nikosev** [comment]
@vardizzo-lab Could you please approve this PR in order to publish these changes?

**Review by vardizzo-lab — APPROVED** [review]
OK.

**Pull request merged**